include/svnlook.php

```diff
@@ -861,6 +861,47 @@ function listFileContents($path, $rev = 0, $peg = '') {
 		}
 	}
 
+	// }}}
+	
+	// {{{ listReadmeContents
+	//
+	// Parse the readme.md file
+	function listReadmeContents($path, $rev = 0, $peg = '') {
+		global $config;
+
+		// Make this Configurable ??
+		$file = "README.md";
+
+		if($this->isFile($path.$file) != True)
+		{
+			return;
+		}
+
+		// Check for the flag for markdown Move this elsewhere?? or better to not added in composer
+		if (!defined('USE_AUTOLOADER')) {
+			require_once 'Parsedown.php';
+		}
+		else {
+			require_once 'include/parsedown.php';
+		}
+		// Capital P and small p to differentiate one used from PEAR or Composer (P) and (p) is from local include
+		$Parsedown = new Parsedown();
+
+		$cmd = $this->svnCommandString('cat', $path.$file, $rev, $peg);
+
+		if ($result = popenCommand($cmd, 'r')) 
+		{
+			echo('<div id="wrap">');
```

Comment:
Is that `id` used for similar rendered code already. Otherwise it might be a good idea to add a more descriptive CSS-class. By using a distinct class it might be possible for customized skins to simply hide those parts easily. Something like `md-rendered`, `rendered-md` or alike?

Reply:
Okay yes this is a good idea. Honestly didnt think of this.  I will do this and get back.

Reply:
The "wrap" is an existing id for rendered content.This is something common across all 3 styles. I tried to copy the same style and rename it to `md-render` or something but seems something is always missed out. Formatting goes away. I guess for now i will keep it as id "wrap" only. This wrap has all definition that will be required for this markdown render like table styles and all. I suggest to keep this for now and lets see if some issue arise will tackle then.